docs/build/changelog.rst

```diff
@@ -25,7 +25,7 @@ Changelog
 
 
     .. change::
-        :tags: bug, commmands
+        :tags: bug, commands
```

Comment:
some projects don't like changing changelogs, but here it seems it could be valuable to fix

Reply:
Should be ok